packages/gatsby-admin/src/components/recipes-gui/index.js

```diff
@@ -10,7 +10,7 @@ import { createUrqlClient } from "../../urql-client"
 import { useMutation, useSubscription } from "urql"
 
 import lodash from "lodash"
-import fetch from "isomorphic-fetch"
+import fetch from "cross-fetch"
```

Comment:
Why switch to cross-fetch?

```suggestion
import fetch from "cross-fetch"
```

Reply:
[From the cross-fetch README:](/lquixada/cross-fetch#why-not-isomorphic-fetch)

Why not isomorphic-fetch?
My preferred library used to be isomorphic-fetch but it has this bug that prevents it from running in a react native environment. It seems unlikely to be fixed since there haven't been any new commits to it since 2016. That means dependencies are outdated as well.

Reply:
Additionally, `isomorphic-fetch` uses `node-fetch@1`, which is vulnerable to CVE-2020-15168 (disclosed yesterday).

Reply:
In saying that, `cross-fetch` uses exact version constraints so is also currently affected by the same advisory :/

Reply:
Pulled out this security fix into its own PR here: https://github.com/gatsbyjs/gatsby/pull/26876

Reply:
Thank you @karlhorky! Beat me to it haha